bin/go-rpm-integration

```diff
@@ -218,7 +218,7 @@ done
 
 provides() {
 goipath="${1}"
-for prov in $(\
+for prov in ${goipath} $(\
```

Comment:
If the root goipath does not provide any Go files, it should not be provided by the devel rpm. Is this the case?

Reply:
Symlink autodeps need a stable anchor and humans expect BR  golang(canonical import path) to work

So even though it may have no go files inside the import path root is special, the rest of the directory structure does not work in all case without it

Reply:
> Symlink autodeps need a stable anchor

Can you provide example and how it is/can be used in practise?

> humans expect BR golang(canonical import path) to work

I am human and I don't expect `BR golang(canonical import path)` to work as if it does not contain any *.go files, it can not be imported in any valid Go code.

Reply:
It's all in commit df69bd558eb9d9ca6c3bb67e5de4ee0cdbe02088, try to make that work without something to anchor the requires to